inc/load.php

```diff
@@ -118,6 +118,16 @@ function load_autoload($name){
         return true;
     }
 
+    // plugin namespace
+    if(substr($name, 0, 7) == 'plugin/') {
+        $name = str_replace('/test/', '/_test/', $name); // no underscore in test namespace
```

Comment:
Every (sub)folder named `test` is renamed to `_test`, isn't it? Is that desired?